dev/InfoBar/InfoBar.xaml

```diff
@@ -0,0 +1,203 @@
+
+                            <Grid.ColumnDefinitions>
+                                <ColumnDefinition Width="Auto"/> <!-- Icon -->
+                                <ColumnDefinition Width="*"/>    <!-- Title, message, and action -->
+                                <ColumnDefinition Width="Auto"/> <!-- Close button -->
+                            </Grid.ColumnDefinitions>
+
+                            <Grid.RowDefinitions>
+                                <RowDefinition Height="Auto"/>
+                                <RowDefinition Height="Auto"/>
+                            </Grid.RowDefinitions>
+
+                            <FontIcon x:Name="StandardIcon"
+                                Grid.Column="0"
+                                Visibility="Collapsed"
+                                VerticalAlignment="Top"
+                                Margin="{StaticResource InfoBarIconMargin}"
+                                FontSize="{StaticResource InfoBarIconFontSize}"
+                                Glyph="{StaticResource InfoBarInformationalIconGlyph}"
+                                FontFamily="{ThemeResource SymbolThemeFontFamily}"
+                                AutomationProperties.AccessibilityView="Raw" />
+
+                            <Border x:Name="UserIconBorder"
+                                Grid.Column="0"
+                                Visibility="Collapsed"
+                                VerticalAlignment="Top"
+                                Margin="{StaticResource InfoBarIconMargin}"
+                                Child="{Binding RelativeSource={RelativeSource TemplatedParent}, Path=TemplateSettings.IconElement}" />
+
+                            <primitives:InfoBarPanel
+                                Grid.Column="1"
+                                Margin="{StaticResource InfoBarPanelMargin}"
+                                primitives:InfoBarPanel.HorizontalMargin="{StaticResource InfoBarPanelHorizontalLayoutMargin}"
+                                primitives:InfoBarPanel.VerticalMargin="{StaticResource InfoBarPanelVerticalLayoutMargin}">
+
+                                <TextBlock x:Name="Title"
+                                    Grid.Column="0"
+                                    Text="{TemplateBinding Title}"
+                                    primitives:InfoBarPanel.HorizontalMargin="{StaticResource InfoBarTitleHorizontalLayoutMargin}"
+                                    primitives:InfoBarPanel.VerticalMargin="{StaticResource InfoBarTitleVerticalLayoutMargin}"
+                                    TextWrapping="WrapWholeWords"
+                                    AutomationProperties.LandmarkType="Navigation"
+                                    FontWeight="{StaticResource InfoBarTitleFontWeight}"
+                                    FontSize="{StaticResource InfoBarTitleFontSize}"/>
+
+                                <TextBlock x:Name="Message" 
```

Comment:
Same as for the Title TextBlock: Could we add (character) text trimming here as well?

Reply:
Microsoft's text guidelines ever since Metro, has opted to cut off text, rather than applying a Text Trimming ellipses.

I think this should change, but it may be decided not to change this for the control.

https://docs.microsoft.com/en-us/windows/uwp/design/style/typography#clipping-and-ellipses

Reply:
Hmm, I see. Well, I guess if that is the official recommendation for now than we should ship this control following that recommendation.